spec/robots/assembly/jp2_create_spec.rb

```diff
@@ -497,8 +497,8 @@
                                                   administrative: { publish: false, sdrPreserve: false, shelve: false } },
                                                 { type: 'https://cocina.sul.stanford.edu/models/file',
                                                   externalIdentifier: 'https://cocina.sul.stanford.edu/file/661f8462-e0c2-49c4-b617-4d5c11a969b7',
-                                                  label: 'hh222cc3333_05_001.jp2',
-                                                  filename: 'hh222cc3333_05_001.jp2',
+                                                  label: 'hh222cc3333_00_001.jp2',
+                                                  filename: 'hh222cc3333_00_001.jp2',
```

Comment:
this test was failing without this change.  I think it's because of how the mocking was set up, but this was the easiest way to get it working.